tests/FactSet.AnalyticsAPI.Engines.Test/Api/CommonParameters.cs

```diff
@@ -38,5 +38,7 @@ public static class CommonParameters
         public const string DefaultDatesDrequency = "Monthly";
         public const string DefaultDatesAccount = "CLIENT:/BISAM/REPOSITORY/QA/SMALL_PORT.ACCT";
         public const string DefaultLookupDirectory = "client:";
+        public static readonly string Quant_Custom_Max_Age = !string.IsNullOrEmpty(Environment.GetEnvironmentVariable("QUANT_CUSTOM_MAX_AGE")) 
+            ? Environment.GetEnvironmentVariable("QUANT_CUSTOM_MAX_AGE") : "5";
```

Comment:
Please declare the values as constants. This applies to the test cases as well.

Reply:
there is no constant file, all constants are declared in this file only.